slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/SlashingProtectionFactory.java

```diff
@@ -22,16 +23,38 @@
 
 public class SlashingProtectionFactory {
 
+  public static final int EXPECTED_DATABASE_VERSION = 6;
+  private static final String errorMsg =
+      String.format(
+          "Database does not have expected version (%s), please run migrations and try again.",
```

Comment:
Can we also include the current db version in the message

Reply:
No - coz we don't know if it was there or not :( (might be empty table)